examples/SEP-24/src/main/kotlin/org/stellar/walletsdk/Main.kt

```diff
@@ -0,0 +1,122 @@
+package org.stellar.walletsdk
+
+import java.math.BigDecimal
+import kotlin.time.Duration.Companion.seconds
+import kotlinx.coroutines.delay
+import org.stellar.sdk.KeyPair
+import org.stellar.sdk.Network
+import org.stellar.sdk.Server
+import org.stellar.sdk.Transaction
+
+// Setup main account that will fund new (user) accounts. You can get new key pair and fill it with
+// testnet tokens at
+// https://laboratory.stellar.org/#account-creator?network=test
+private val myAddress = "GCNVZJLJ5XZZALLZBR7BGUMGMF2KVZTBIGTU2A5KOPS2PPCZS6PV4LAD"
+private val myKey = "SDYGC4TW5HHR5JA6CB2XLTTBF2DZRH2KDPBDPV3D5TXM6GF7FBPRZF3I"
+
+suspend fun main() {
+  // Create instance of server that allows to connect to Horizon
+  val server = Server("https://horizon-testnet.stellar.org")
+  val wallet = Wallet()
```

Comment:
Just a heads up, the API for `Wallet` changes slightly in [this PR](https://github.com/stellar/kotlin-wallet-sdk/pull/21).